Dashboard/app/js/lib/views/surveys/question-view.js

```diff
@@ -197,6 +206,16 @@ FLOW.QuestionView = FLOW.View.extend({
     	FLOW.selectedControl.set('selectedCascadeResource', cascadeResource);
     }
 
+    // reset selected caddisfly resource
+    FLOW.selectedControl.set('selectedCaddisflyResource', null);
+    // if the caddisflyResourceUuid is not null, get the resource
+    if (!Ember.empty(FLOW.selectedControl.selectedQuestion.get('caddisflyResourceUuid'))) {
+      FLOW.caddisflyResourceControl.get('content').forEach(function (item) {
+        if (item.get('uuid') == FLOW.selectedControl.selectedQuestion.get('caddisflyResourceUuid')) {
+          FLOW.selectedControl.set('selectedCaddisflyResource', item);
```

Comment:
Possibly, but I don't have time to find out. There are only about 10 of them or so.